editor/state-machine/inputs.mdx

```diff
@@ -3,7 +3,13 @@ title: 'Inputs'
 description: ''
 ---
 
-Inputs are the contract between designers and developers. As Designers, we use Inputs as a way to control the transitions in our state machine by assigning them as conditions. Developers tie into the inputs at runtime and define conditions through code that can change those inputs. 
+<Warning>
+  We strongly recommend controlling your Rive file through [Data Binding](../data-binding), rather than Inputs. Not only is Data Binding more powerful, it's easier to [control via code at runtime](../../runtimes/data-binding).
+
+  Data Binding includes all the functionality of Inputs, as well as the ability to control colors, enums, and strings.
```

Comment:
We can remove this sentence. Data binding does even more than that, and you already sold it well above "more powerful and easier to control"